README.md

```diff
@@ -1,4 +1,35 @@
-# Safe Apps SDK
+# Gnosis Safe apps SDK
+
+[![Logo](https://raw.githubusercontent.com/gnosis/safe-apps-sdk/master/assets/logo.png)](https://gnosis.pm/)
+
+[![Build Status](https://travis-ci.org/gnosis/safe-apps-sdk.svg?branch=master)](https://travis-ci.org/gnosis/pm-contracts)
+
+Software development kit to integrate a third-party app with Safe Multisig (https://gnosis-safe.io/app/).
 
 ## Install
-- `yarn`
+
+### Install requirements with yarn:
+
+```bash
+yarn install @gnosis.pm/safe-app-sdk
+```
+
+## Build
+
+```bash
+yarn build
+```
+
+## Documentation
+
+Documentation can be found in [docs.gnosis.io](https://docs.gnosis.io/safe/docs/docs5/)
```

Comment:
that was changes just today :P

Reply:
it's a new link? anyways I'll remove that line until we have the doc there. For now I'll add the doc in Readme.md

Reply:
https://docs.gnosis.io/safe/docs/sdks_safe_apps/